ukrflix/pom.xml

```diff
@@ -136,6 +142,11 @@
             <artifactId>jackson-databind</artifactId>
             <version>2.12.4</version>
```

Comment:
1)please extract all dependencies versions into maven properties
https://stackoverflow.com/questions/37358318/maven-dependency-version-as-property
2) please don't forget to use dependenciesManager where we need it
https://www.baeldung.com/maven-dependencymanagement-vs-dependencies-tags